senza.yaml

```diff
@@ -0,0 +1,162 @@
+SenzaInfo:
+  StackName: "{{Arguments.StackName}}"
+  Parameters:
+    - DockerImage:
+        Description: "Name of the Docker image providing Nakadi."
+    - StackName:
+        Description: "Name of the Cloud Formation Stack"
+    - AccessTokenUrl:
+        Description: "Url to the access token validation service"
+    - TokeninfoUrl:
+        Description: "TokeninfoUrl"
+    - ApplicationId:
+        Description: "Application id"
+        Default: "nakadi"
+    - MintBucket:
+        Description: "Name of the mint bucket to use for this application"
+    - ScalyrAccountKey:
+        Description: "Key for Skalyr logging (optional)"
+        Default: ""
+    - InstanceType:
+        Description: "The EC2 Instance type that is used for the Nakadi instances (optional, default: t2.micro)"
+        Default: "t2.micro"
+    - InstancesMinumum:
+        Description: "The minimum number of instances of the AutoScalingGroup (optional, default: 2)"
+        Default: 2
+    - InstancesMaximum:
+        Description: "The maximum number of instances of the AutoScalingGroup (optional, default: 5)"
+        Default: 5
+    - ZookeeperKafkaNamespace:
+        Description: "Namespace in Zookeeper used by Buku (Kafka) (optional)"
+        Default: ""
+    - ExhibitorBrokers:
+        Description: "List of exhibitor brokers (comma separated)"
+    - ExhibitorPort:
+        Description: "Exhibitor port (ptional, default 8181)"
+        Default: "8181"
+    - LimitTrafficToCidr:
+        Description: "Allows request to the loadbalancer of Nakadi to only this ip address range given in cidr notation (optional, default: 0.0.0.0/0)"
+        Default: "0.0.0.0/0"
+
+SenzaComponents:
+
+  - Configuration:
+      Type: Senza::StupsAutoConfiguration # auto-detect network setup
+
+  - AppServer:
+      Type: Senza::TaupageAutoScalingGroup
+      InstanceType: "{{Arguments.InstanceType}}"
+      SecurityGroups:
+        - "Fn::GetAtt":
+          - "NakadiSecurityGroup"
+          - "GroupId"
+      IamInstanceProfile:
+        Ref: NakadiIamInstanceProfile
+      ElasticLoadBalancer: AppLoadBalancer
+      AssociatePublicIpAddress: false
+      TaupageConfig:
+        application_id: "{{Arguments.ApplicationId}}"
+        application_version: "{{SenzaInfo.StackVersion}}"
+        mint_bucket: "{{Arguments.MintBucket}}"
+        runtime: Docker
+        source: "{{Arguments.DockerImage}}"
+        health_check_path: /health
+        ports:
+          8080: 8080
+        scalyr_account_key: "{{Arguments.ScalyrAccountKey}}"
+        environment:
```

Comment:
- Some of these are legacy variables from python version which are not used in Nakadi-jvm

- Also I think it would be good to take a look at the latest list of properties https://github.com/zalando/nakadi/blob/nakadi-jvm/src/main/resources/application.yml and to check whether we want to expose some of that here. I believe at least the replica factor should be set here (because in default config it is 1)